types/views/views.go

```diff
@@ -170,6 +170,23 @@ func SliceContains[T comparable](v Slice[T], e T) bool {
 	return false
 }
 
+// SliceEquals reports whether a and b contain the same elements, regardless of order.
+func SliceEquals[T comparable](a, b Slice[T]) bool {
+	if a.Len() != b.Len() {
+		return false
+	}
+	aHasValue := make(map[T]bool)
+	for i, n := 0, a.Len(); i < n; i++ {
+		aHasValue[a.At(i)] = true
```

Comment:
are [A, B, B, C, C, C]
and [A, B, B, B, C, C] equal?

Under this implementation, they are. But that's kinda weird.

We really want to count the number of occurrences of each.